File: Services/User/User.Domain/User_Report.cs
using System.ComponentModel.DataAnnotations;

namespace User.Domain;

public class User_Report
{
    public int user_id { get; set; }
    public int report_id { get; set; }
    public static User_Report Create(int user_id,int report_id)
    {
        return new User_Report
        {
            user_id = user_id,
            report_id = report_id
        };
    }
}

File: Services/DB_Handler/models/UserReport.py
from sqlalchemy import Column, Integer, ForeignKey
from sqlalchemy.orm import relationship
from .base import Base


class UserReport(Base):
    __tablename__ = 'user_report'

    user_id = Column(Integer, ForeignKey("users.user_id"), primary_key=True)
    review_id = Column(Integer, ForeignKey("report.report_id"), primary_key=True)

    user = relationship("Users", back_populates="user_reports")
    report = relationship("Report", back_populates="user_reports")
